spec/acceptance/supportedos_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'openvmtools class' do
+  context 'default parameters' do
+    it 'work with no errors' do
+      pp = <<-EOS
+        class { 'openvmtools': }
+      EOS
+
+      apply_manifest(pp, catch_failures: true)
+    end
```

Comment:
We have https://github.com/voxpupuli/voxpupuli-acceptance/?tab=readme-ov-file#an-idempotent-resource and you can also write that shorter:
```suggestion
    it_behaves_like 'an idempotent resource' do
      let(:manifest) { 'include openvmtools' }
    end
```

Reply:
eh, I know but it not run indempotent, because the service not start causing a systemd unit check

Reply:
Could you then leave that as a comment in the file why it's not being checked for idempotency?

Reply:
Done